code/game/objects/items/airlock_painter.dm

```diff
@@ -384,7 +384,7 @@
 		list("White", "#D4D4D4"),
 		list("Black", "#0e0f0f"),
 		list("Bar Burgundy", "#791500"),
-		list("Sec Red", "#DE3A3A"),
+		list("Sec Blue", "#486091"), // SKYRAT EDIT CHANGE
```

Comment:
Might be worth leaving a Red in for non-sec red decals, just a generic "Red"

Reply:
You can still set a custom colour of anything you choose, so I think it's probably fine to just replace it straight up